config/detekt.yml

```diff
@@ -60,7 +57,7 @@ exceptions:
   ReturnFromFinally:
     active: true
   SwallowedException:
-    active: true
+    active: false
```

Comment:
i know there's a couple places where we're suppressing this exception and it is not pretty, but i think we want this check for when we're accidentally eating an exception without doing something about it